lms/static/sass/shared-v2/_components.scss

```diff
@@ -149,7 +149,7 @@
 
   margin: 0 auto;
   padding: ($baseline*0.75) ($baseline*2);
-  background-color: $lms-preview-menu-color;
```

Comment:
This might not apply the same background color to all three styles (v1, v2, and v3). If we think we want it to be the primary color, we should just remove the variable entirely.